cmd/swarm/cleandb.go

```diff
@@ -0,0 +1,23 @@
+package main
```

Comment:
Please put a newline between the header and package declaration to prevent showing the license header text in GoDoc.